src/ledger/test/LedgerTxnTests.cpp

```diff
@@ -145,7 +149,8 @@ TEST_CASE("LedgerTxn commit into LedgerTxn", "[ledgertxn]")
     VirtualClock clock;
     auto app = createTestApplication(clock, getTestConfig());
 
-    LedgerEntry le1 = LedgerTestUtils::generateValidLedgerEntry();
+    LedgerEntry le1 = LedgerTestUtils::generateValidLedgerEntryWithExclusions(
```

Comment:
I'm concerned about skipping a lot of tests involving `SpeedexConfigurationEntry` only because it can't be erased. I see two real options here

- use conditions around the `erase` cases to avoid them while executing the rest
- actually allow `SpeedexConfigurationEntry` to be erased at the `LedgerTxn` level (this approach isn't unprecedented: claimable balances are immutable but we still allow them to be modified at the `LedgerTxn` level)

Reply:
Moved the speedex exclusion directly to cases that call 'erase' or generate dead ledger entries. The tests that cover erase only still omit speedex for all the cases.